queries.sql

```diff
@@ -689,30 +689,27 @@ campLists AS (
     WHERE campaign_lists.campaign_id = $1
 ),
 subIDs AS (
-    SELECT DISTINCT ON (subscriber_lists.subscriber_id) subscriber_id, list_id, status FROM subscriber_lists
+    SELECT DISTINCT ON (sl.subscriber_id) sl.subscriber_id, sl.list_id, sl.status FROM subscriber_lists sl
+    INNER JOIN subscribers ON (subscribers.id = sl.subscriber_id AND subscribers.status!='blocklisted')
```

Comment:
Instead of doing a join on subscribers here and another later, it should ideally be just one.

Reply:
I remade it. Please take a look. 

Also i removed filter campLists.optin = 'double' in opt-in campaign. 
I think that this is logical mistake.